src/main/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -689,13 +689,15 @@ private void discoverOres(final ItemStack stack)
     {
         if (stack.is(Tags.Items.ORES) || stack.is(ModTags.breakable_ore) || stack.is(ModTags.raw_ore))
         {
-            if (stack.getItem() instanceof BlockItem)
+            final ItemStack smeltingResult = MinecoloniesAPIProxy.getInstance().getFurnaceRecipes().getSmeltingResult(stack);
+            if (smeltingResult != null && !ItemStackUtils.compareItemStacksIgnoreStackSize(smeltingResult, stack))
```

Comment:
The code is reversed, so it checks for a smelting recipe first, and marks it as being broken in other cases

Which solves the issue for raw redstone ore and ancient debris, but will probably give issues for regular ore blocks

Reply:
It's just a sanity check, so we don't add sth that just becomes itself here as a smeltable item. The if is still checked for both items/blocks

Reply:
> The code is reversed, so it checks for a smelting recipe first, and marks it as being broken in other cases

The reversed order makes no difference, because the two checks are not exclusive and operate on different output collections.

Reply:
I missed that the `else` was removed, which solved the issue for regular ores

Reply:
It wasn't removed; it was never there.  Hence why I'm saying that this code effectively did not change behaviour at all.

Reply:
I added an else temporarily there, but removed it afterwards. This code change just avoids blocks with buggy smelting recipes from being added as a smeltable ore.

Reply:
I happened to see the version with the temporary `else`